### PR TITLE
New avx512_vnni kernel

### PR DESCRIPTION
**Pull request by luoyu-intel**
## Type of Change

1. update jblas
2. Use the new avx512_vnni kernel to get a higher first token perf.

## How has this PR been tested?

llama-7B bits=4 group_size=128, 32 comp_type=int8

### CODE REVIEW

**Comment by luoyu-intel** [comment]
q4 group_size=128,  perf on xeon 6248 20cores:

before: 
perf_total_per_op_us[                 MUL_QKV] = 1630.308 ms
perf_total_per_op_us[                FFN_SILU] = 4392.396 ms

after:
perf_total_per_op_us[                 MUL_QKV] = 917.410 ms
perf_total_per_op_us[                FFN_SILU] = 2533.962 ms

File: intel_extension_for_transformers/llm/library/jblas/jblas/jit_blas_weight_compression.h [hunk 1]
```diff
@@ -394,7 +371,8 @@ class WeightS8ScaleFp32 {
     auto ptr = dynamic_cast<PackedWeightKBlock*>(stor);
     if (ptr) {
       int nk_scale = utils::updiv(K, ptr->mBlockSize);
-      StorageSimpleCorrection<float, int8_t> corr(N * nk_scale, is_sym);
+      StorageSimpleCorrection<float, int8_t, float> corr;
```
Comment (airMeng) [inline intel_extension_for_transformers/llm/library/jblas/jblas/jit_blas_weight_compression.h:374]:
```suggestion
      using CorrectionType = StorageSimpleCorrection<float, int8_t, float>;
      CorrectionType  corr;
```

align with others

**Review by airMeng — APPROVED** [review]

**Comment by luoyu-intel** [comment]
The new kernel introduces an extra reduce buffer(same size as scale). This will slow down the next token inference.
So, the next token generation will use the previous kernel without the reduce buffer.

File: intel_extension_for_transformers/llm/library/jblas/jblas/jit_blas_weight_compression.h (hunk 1 above)
Comment (luoyu-intel) [reply intel_extension_for_transformers/llm/library/jblas/jblas/jit_blas_weight_compression.h:374]: will be aligned in jblas repo.

**Comment by luoyu-intel** [comment]
q4 group_size=128, perf on xeon 8280 28cores:

before:
perf_total_per_op_us[                 MUL_QKV] = 1176.466 ms
perf_total_per_op_us[                FFN_SILU] = 3134.920 ms

after:
perf_total_per_op_us[                 MUL_QKV] = 659.539 ms
perf_total_per_op_us[                FFN_SILU] = 1788.060 ms

**Review by a32543254 — APPROVED** [review]
LGTM

**Review by zhewang1-intc — APPROVED** [review]

**Pull request merged**